python/cudf/cudf/core/frame.py

```diff
@@ -304,6 +304,44 @@ def _unaryop(self, op):
             result._data[name] = col.unary_operator(op)
         return result
 
+    def isnull(self):
+        """Identify missing values.
+        """
+        return self._is_null()
+
+    def isna(self):
+        """Identify missing values. Alias for `isnull`
+        """
+        return self._is_null()
+
+    def notnull(self):
+        """Identify missing values.
+        """
+        return self._is_valid()
+
+    def notna(self):
+        """Identify non-missing values. Alias for `notnull`.
+        """
+        return self._is_valid()
+
+    def _is_null(self):
+        result = self.copy()
+        for name, col in result._data.items():
+            result._data[name] = col.isnull()
```

Comment:
This feels a bit awkward to have to reach into the ColumnAccessor here. @shwina is this the expected way to handle this from `Frame`?

Reply:
Yeah I’d rather have a libcudf api accepting a table_view, and use the established pattern, but thats not available in this case. Do we just assemble the table on this side, instead?

Reply:
I think what would be less awkward is constructing an `OrderedColumnDict`: https://github.com/rapidsai/cudf/blob/branch-0.13/python/cudf/cudf/utils/utils.py#L359 (can init the same as a normal `OrderedDict` but enforces types / lengths)

Then pass that into the `Table` init as `data` along with `self.index`: https://github.com/rapidsai/cudf/blob/branch-0.13/python/cudf/cudf/_libxx/table.pyx#L31

Maybe a little bit more verbose but feels more explicitly like the behavior you're trying to accomplish.

Reply:
I'm pushing my changes so I can compare them better, but I still need to make changes, so feel free to ignore for now.

Reply:
Changes look good, resolving